cms/djangoapps/contentstore/management/commands/backfill_course_tabs.py

```diff
@@ -50,4 +53,9 @@ def handle(self, *args, **options):
                 # This will trigger the Course Published Signal which is necessary to update
                 # the corresponding Course Overview
                 logger.info(f'Updating tabs for {course_key}.')
-                store.update_item(course, ModuleStoreEnum.UserID.mgmt_command)
+                try:
```

Comment:
Does the error logger print the stack even if the exception is eaten? In other words, will this error message be able to indicate what management command was updating the course?

Reply:
`logger.exception()` prints the traceback - so we'll still get it here.

Reply:
Yes, it should. logger.exception includes the stack trace by default. Also it will be in the context of running this command so you will definitely know it was this command that did it.